Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1751,6 +1752,9 @@ void RenderLayerScrollableArea::clearScrollCorner()
 
 void RenderLayerScrollableArea::updateResizerStyle()
 {
+    if (!m_resizer || !m_layer.canResize())
```

Comment:
This seems wrong. The function can create an `m_resizer` below, and maybe `layer.canResize()` changed since last time, requiring us to remove the existing resizer.

Reply:
Hmm. I think "&&" make sense. Let me push the original update. Unless you think this early return in the concept is wrong then I can close this PR.

Reply:
@smfr - `mac-wk1` seems to have issue so this failure is not related to this patch.

Reply:
@smfr - Any update? Or can I close this PR?